exporters/variables-scss/src/content/generators/fileGenerator.ts

```diff
@@ -0,0 +1,56 @@
+import { TokenGroup, Token, TokenType } from '@supernovaio/sdk-exporters';
+import { generateFileContent } from './contentGenerator';
+
+const filesData = [
+  {
+    fileName: '_spaces.scss',
```

Comment:
spacing is better imho

```suggestion
    fileName: '_spacing.scss',
```